beta/src/components/MDX/Sandpack/utils.ts

```diff
@@ -113,10 +112,12 @@ export const useSandpackLint = () => {
   const onLint = linter((props: EditorView) => {
     const editorState = props.state.doc;
     return import('./eslint-integration').then((module) => {
-      let {errors} = module.lintDiagnostic(editorState);
-      // Only show errors from rules, not parsing errors etc
-      setLintErrors(errors.filter((e) => !e.fatal));
-      return module.lintDiagnostic(editorState).codeMirrorPayload;
+      let {errors, codeMirrorErrors} = module.lintDiagnostic(editorState);
+
+      const excludeParsingError = (error: any) => !!error.ruleId;
```

Comment:
Use only errors with a valid `ruleId`, I believe it's the safest way to ensure having only the desired errors.